src/Nest/Cluster/NodesStats/NodeStats.cs

```diff
@@ -571,4 +574,74 @@ public class HttpStats
 		[DataMember(Name = "total_opened")]
 		public long TotalOpened { get; internal set; }
 	}
+
+	[DataContract]
+	public class IndexingPressureStats
+	{
+		[DataMember(Name = "memory")]
+		public IndexingPressureMemoryStats Memory { get; internal set; }
+	}
+
+	[DataContract]
+	public class IndexingPressureMemoryStats
+	{
+		[DataMember(Name = "current")]
+		public IndexingLoad Current { get; internal set; }
+
+		[DataMember(Name = "total")]
+		public TotalIndexingLoad Total { get; internal set; }
+
+		[DataMember(Name = "limit_in_bytes")]
+		public long LimitInBytes { get; internal set; }
+
+		[DataMember(Name = "limit")]
+		public string Limit { get; internal set; }
+	}
+
+	[DataContract]
+	public class IndexingLoad
+	{
+		[DataMember(Name = "combined_coordinating_and_primary_in_bytes")]
+		public long CombinedCoordinatingAndPrimaryInBytes { get; internal set; }
+
+		[DataMember(Name = "combined_coordinating_and_primary")]
+		public string CombinedCoordinatingAndPrimary { get; internal set; }
+
+		[DataMember(Name = "coordinating_in_bytes")]
+		public long CoordinatingInBytes { get; internal set; }
+
+		[DataMember(Name = "coordinating")]
+		public string Coordinating { get; internal set; }
+
+		[DataMember(Name = "primary_in_bytes")]
+		public long PrimaryInBytes { get; internal set; }
+
+		[DataMember(Name = "primary")]
+		public string Primary { get; internal set; }
+
+		[DataMember(Name = "replica_in_bytes")]
+		public long ReplicaInBytes { get; internal set; }
```

Comment:
Does this always have a value? if not should be `long?`.

Reply:
In my tests against a local 7.10 node, it always returns as 0 so I assume safe to make it non nullable.

Reply:
![image](https://user-images.githubusercontent.com/3669103/101175073-9f17b200-363c-11eb-8768-b7e504ac7b15.png)